### PR TITLE
feat(client): add social metas

### PR DESCRIPTION
**Pull request by evenchange4**
Add metas to fix something like:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,6 +5,22 @@
   <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
   <meta content="utf-8" http-equiv="encoding">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="description" content="Try PureScript is an online PureScript code editor for quickly experimenting with PureScript code snippets and ideas.">
+
+  <!--  Facebook  -->
+  <meta property="og:type" content="website">
+  <meta property="og:url" content="http://try.purescript.org/">
+  <meta property="og:title" content="Try PureScript!">
+  <meta property="og:image" content="https://raw.githubusercontent.com/purescript/logo/master/youtube/channel_icon-800.png">
```
Comment (paf31) [inline index.html]: Maybe we should copy this image into this repo?

**Comment by paf31** [comment]
Great, thanks!

File: index.html (hunk 1 above)
Comment (evenchange4) [reply index.html]: Done. 🙌

File: index.html [hunk 2]
```diff
@@ -5,6 +5,22 @@
   <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
   <meta content="utf-8" http-equiv="encoding">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="description" content="Try PureScript is an online PureScript code editor for quickly experimenting with PureScript code snippets and ideas.">
+
+  <!--  Facebook  -->
+  <meta property="og:type" content="website">
+  <meta property="og:url" content="http://try.purescript.org/">
+  <meta property="og:title" content="Try PureScript!">
+  <meta property="og:image" content="http://try.purescript.org/img/channel_icon-800.png">
```
Comment (paf31) [inline index.html]: One last thing - could you please add an attribution notice in a comment here? Something like

```html
<!-- 
The PureScript Logo by Gareth Hughes is licensed under the Creative Commons Attribution 4.0 license (https://creativecommons.org/licenses/by/4.0/).
-->
```

Thanks!

Comment (paf31) [reply index.html]: Or maybe you could use `twitter:image:alt`.

Comment (evenchange4) [reply index.html]: Sure, we should add this info!

**Comment by hdgarrood** [comment]
Thanks!

**Pull request merged**